apps/rocketcleaner/rocketcleaner.yml

```diff
@@ -0,0 +1,11 @@
+name: RocketCleaner
+description: 'MAC OS hdd & ram.'
```

Comment:
These description doesn't answer for questions "what's, why" this application does.

Reply:
Ummm Ok but this is not my app, So I will ask @miladdavoodi to answer that and will make the changes

Reply:
Do you mean my app doesn't work?
@HashimotoYT 

@Sanjit1 
#After running the program performs the following tasks

# Empty the Trash on all mounted volumes and the main HDD..
- rm -rfv /Volumes/*/.Trashes/* &>/dev/null
- rm -rfv ~/.Trash/* &>/dev/null
# Clear System Log Files...
- rm -rfv /private/var/log/asl/*.asl &>/dev/null
- rm -rfv /Library/Logs/DiagnosticReports/* &>/dev/null
- rm -rfv /Library/Logs/Adobe/* &>/dev/null
- rm -rfv ~/Library/Containers/com.apple.mail/Data/Library/Logs/Mail/* &>/dev/null
- rm -rfv ~/Library/Logs/CoreSimulator/* &>/dev/null
# npm cache clean
- npm cache clean --force
# Clear Adobe Cache Files...
- rm -rfv ~/Library/Application\ Support/Adobe/Common/Media\ Cache\ Files/* &>/dev/null
# Cleanup iOS Applications...
- rm -rfv ~/Music/iTunes/iTunes\ Media/Mobile\ Applications/* &>/dev/null
# Remove iOS Device Backups...
- rm -rfv ~/Library/Application\ Support/MobileSync/Backup/* &>/dev/null
# Cleanup XCode Derived Data and Archives...
- rm -rfv ~/Library/Developer/Xcode/DerivedData/* &>/dev/null
- rm -rfv ~/Library/Developer/Xcode/Archives/* &>/dev/null
# Cleanup pip cache...
- rm -rfv ~/Library/Caches/pip

#Purge inactive memory... 
- purge

Reply:
No. It's just that we need a better description. The description does not answer the question: What does this app do?
or
Why would someone need this app?

But I think you answered the question. So I will go with the description to :
`Empty trash from HDD's, Cleans system log files, IOS Applications, pip Cache, etc.`

[![https://img.shields.io/badge/Keep%20Koding%2C%20Best%20regards-Sanjit-accff2](https://img.shields.io/badge/Best%20regards-Sanjit-accff2?style=for-the-badge)](https://www.sanjit.wtf)

Reply:
thank you @Sanjit1